src/library_dylink.js

```diff
@@ -250,8 +250,8 @@ var LibraryDylink = {
 
   // Loads a side module from binary data. Returns the module's exports or a
   // progise that resolves to its exports if the loadAsync flag is set.
-  $loadSideModule__deps: ['$loadDynamicLibrary', '$createInvokeFunction', '$getMemory', '$relocateExports', '$resolveGlobalSymbol', '$GOTHandler'],
-  $loadSideModule: function(binary, flags) {
+  $loadWebAssemblyModule__deps: ['$loadDynamicLibrary', '$createInvokeFunction', '$getMemory', '$relocateExports', '$resolveGlobalSymbol', '$GOTHandler'],
```

Comment:
@sbc100 I noticed that this `$loadWebAssemblyModule__deps__deps` array contains `$loadDynamicLibrary`, which seems odd because `loadDynamicLibrary` depends on `loadWebAssemblyModule`, not the other way around. Are these `__deps` arrays meant to list a function's direct dependencies, or are they expressing a different relationship than that?

Reply:
Just direct dependencies yes.   

AFAICT, loadWebAssemblyModule does call loadDynamicLibrary.  It uses it to load the libraries dependencies down near the end of the function, doesn't it?   So it seems they are mutually dependent

Reply:
Oh yeah, I see. Thanks!